_std/_std_readme.txt

```diff
@@ -0,0 +1,10 @@
+if your file is BROADLY meant to just be a list of defines for value replacement, put it in the defines folder.
```

Comment:
Make this into a .md file so it renders on the documentation, please. Probably best to name README too.

Reply:
alright, good catch!